namespaces/kubernetes.yml

```diff
@@ -45,6 +45,17 @@ namespace:
     description: >
       Kubernetes Master URL
 
+  - name: container_name
+    type: string
+    norms:
+      enabled: true
+    description: >
+      The name of the container in Kubernetes.
+    fields:
+      - name: raw
```

Comment:
I don't think you need raw here, as `container_name` itself should be not_analyzed